include/swift/AST/diagnostics/fr.yaml

```diff
@@ -17,3 +17,14 @@
 # 
 #===----------------------------------------------------------------------===#
 
+- id: "lex_unterminated_string"
+  msg: "chaîne non terminée littérale"
```

Comment:
Out of curiosity - how do we validate translations to make sure there’s no mistakes, the original meaning/information is correctly preserved, etc? Is there going to be some sort of document that explains this process?

Reply:
Yes, we have thought about that :) We'll definitely need community support and translations are going to be vetted before inclusion. We are working out details right now at the moment. We'd also build lint and other tools to maintain localizations.

Reply:
TBH, I used google translate. I think we would need professional translators that can translate the messages? I really don't know. Regarding for us to know if the meaning/ information is correct... I don't think there's an easy way for that other than learning French in that case, haha 😅

Reply:
@xedin since the translated messages will be are going to be vetted before including them in the compiler. Should I delete the translations from the main directories for now i.e. `include/swift/AST/diagnostics/fr.yaml` & `en.yaml` and only leave them in the `test/diagnostics/Localization/Inputs` for testing purposes only?

Reply:
Yes, that's a good idea.